README.md

```diff
@@ -1,345 +1,245 @@
-# Veritrans ruby library
+# Midtrans Ruby library
 
```

Comment:
Write guide to maintaining this repo e.g. https://github.com/Midtrans/midtrans-nodejs-client/blob/master/Maintaining.md, at least how to run the test etc for this repo if our other team member want to maintain this package.

Reply:
No need how-to update ruby gem yet if you don't know, since we haven't got access.